library/src/main/java/com/datatorrent/lib/io/fs/AbstractSingleFileOutputOperator.java

```diff
@@ -52,6 +81,7 @@ protected String getFileName(INPUT tuple)
   public void setOutputFileName(String outputFileName)
   {
     this.outputFileName = outputFileName;
+    partitionedFileName = String.format(partitionedFileNameformat, outputFileName, physicalPartitionId);
```

Comment:
My assumption was when we set property values for the operators in the running application using apexcli; then it will only invoke setter method and setup method will not be invoked. 
Hence, I am initializing derived field values in the setter of original property.